code_challenges/code_wars/sort_array.py

```diff
@@ -1,9 +0,0 @@
-def sort_array(numbers_to_sort):
```

Comment:
Why are you deleting this file?

Reply:
im not sure why it ended up here but i already have a brach pushed through onto main of sort array

Reply:
If you delete this file and commit the deletion, when you merge it will be removed from main.

This doesn't seem like you intention.   

1) checkout to main.
2) open this file in vs code and add a single space, don't save but leave the file open
3) check back out to this branch
4) remove the added space from this file and save the file back on this branch.  (Now the file should look exactly like it looks on main.
5) Add and commit this file
6) Push the changes to this branch on Github.

If all goes well, the file will be removed from the diff, since it's already on main.